dev-haskell/clay/clay-0.14.0.ebuild

```
@@ -0,0 +1,29 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.8.4.0.9999
+
+CABAL_CHDEPS=(
+	'base  >= 4.11 && < 4.16' 'base  >= 4.11 && < 5'
+)
```

I'm getting an error in `src_configure`:

```
setup: Encountered missing or private dependencies:
base >=4.11 && <4.16
```

---

Your first `CABAL_CHDEPS` entry doesn't match the `base` dependency in `Test-Suite Test-Clay`:

```cabal
  Build-Depends:
    base                 >= 4.11  && < 4.16,
```

---

```suggestion
CABAL_CHDEPS=(
	'base  >= 4.11 && < 4.16' 'base  >= 4.11 && < 5'
	'base                 >= 4.11  && < 4.16' 'base  >= 4.11 && < 5'
)
```

---

I see, thanks for a fix!